packages/ui-extensions/src/surfaces/admin/api/customer-segment-template/customer-segment-template.doc.ts

```diff
@@ -0,0 +1,21 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'CustomerSegmentTemplate Extension API',
+  featureFlag: 'admin_extensibility',
```

Comment:
nit: Not sure if this feature flag is needed? We are gating behind a different flag in CLI.

Reply:
hmm, all the other API docs are gated behind this flag. I can double check

Reply:
looks like this featureFlag isn't in the generated file in shopify-dev so I think we can leave it out 👍